### PR TITLE
Hello you crazy kids

### PR DESCRIPTION
**Pull request by EricDopp**
I did what Jonathan told me to do.

### CODE REVIEW

**Review by justin-willow — COMMENTED** [review]
Hello!

**Review by EricDopp — COMMENTED** [review]
Okay

File: MidTermProject/Program.cs [hunk 1]
```diff
@@ -1,2 +1 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
+﻿Console.WriteLine("Hello fellow kids");
```
Comment (EricDopp) [inline MidTermProject/Program.cs:1]: That's not how the meme goes Eric!

**Review by justin-willow — APPROVED** [review]
I guess this'll do.

**Pull request merged**